web/yo/app/scripts/controllers/gene.js

```diff
@@ -121,6 +139,7 @@ angular.module('oncokbApp')
              * The other one is about the detailed treatment info inside when first open the tumor section, and the result is stored in tumorContent.
              * **/
             $rootScope.getTumorMessages = function(mutation) {
+                return true;
```

Comment:
This serves to temporarily disable the function. It will be refactored in the review mode related issues since the duplicated tumors are generated through modifying tumors.